app/uk/gov/hmrc/cdsimportsddsfrontend/services/DeclarationXml.scala

```diff
@@ -230,6 +245,10 @@ object DeclarationXml {
 
   }
 
+  private def maybeElement(name: String, maybeValue: Option[String]) = {
```

Comment:
We need some unit test coverage for this helper

Reply:
It's a private method and therefore not exposed to the user - just used to get round the None, empty space issue on the frontend. This should not really be necessary if the validation worked correctly.
I could actually get at it by using invokePrivate - but it's overkill for temporary solution.